src/api/app/views/webui2/webui/request/_sourcediff_tab.html.haml

```diff
@@ -0,0 +1,37 @@
+%div
+  %b
+    = request_action_header(action, request_creator)
+  -# FIXME: request forwarding # here or decision box?
```

Comment:
??? :eyes:

Reply:
The whole feature is WIP as you know. That's two features that are part of the diff, but also require some work on the decision box.
I'll remove it.

Reply:
I just didn't understand the coment